external/storm-druid/src/main/java/org/apache/storm/druid/bolt/DruidBeamBolt.java

```diff
@@ -76,30 +77,33 @@ public void prepare(Map stormConf, TopologyContext context, OutputCollector coll
 
     @Override
     public void execute(final Tuple tuple) {
-      Future future = tranquilizer.send((druidEventMapper.getEvent(tuple)));
-        LOG.debug("Sent tuple : [{}]" , tuple);
+        if (TupleUtils.isTick(tuple)) {
```

Comment:
Most of these connector bolts extend `BaseRichBolt`. It may be good to have a class extending BaseRichBolt and handle ticktuples. This class can be used across other connector bolts instead of handling them in respective connector's base classes.

```
// you can have better class name if found any
public class BaseRichBoltWithTickTuples extends BaseRichBolt {
  @Override
  public void execute(final Tuple tuple) {
    if(TupleUtils.isTick(tuple)) {
      onTickTuple(tuple);
    } else {
      process(tuple);
    }
  }

  protected void onTickTuple(final Tuple tuple) {
  }

}

```